src/njs_parser.c

```diff
@@ -4606,16 +4605,37 @@ njs_parser_statement(njs_parser_t *parser, njs_lexer_token_t *token,
         break;
     }
 
-    ret = njs_parser_statement_wo_node(parser, token, current);
```

Comment:
While this change seems useful and makes the code identical to other similar places, the change itself is unrelated to the fix.

Reply:
Same as about next comment.